index.d.ts

```diff
@@ -18,12 +18,12 @@ export type LocaleData = {
 
 
 export function registerLocale(localeData: LocaleData): void;
-export function alpha2ToAlpha3(alpha2: string | Alpha2Code): string;
-export function alpha2ToNumeric(alpha2: string | Alpha2Code): string;
-export function alpha3ToAlpha2(alpha3: string | Alpha3Code): string;
-export function alpha3ToNumeric(alpha3: string | Alpha3Code): string;
-export function numericToAlpha2(numeric: number | string): string;
-export function numericToAlpha3(numeric: number | string): string;
+export function alpha2ToAlpha3(alpha2: string | Alpha2Code): string | undefined;
```

Comment:
I'm actually curious why it was decided to put `string` instead of `Alpha3Code` in this one.
Same for other similar cases where we know that the output should be of the given type that is defined later on this file.

@michaelwittig, do you have context on this or know someone who does? Thanks!

Reply:
Sorry, no. I'm not using TypeScript.

Reply:
@alias-mac Can I still merge this without the `Alpha3Code` change you mentioned?

Reply:
Yes, I think we can make further improvements later while this PR is still valid from a return type perspective.